src/products/components/ProductVariantEndPreorderDialog/ProductVariantEndPreorderDialog.tsx

```diff
@@ -0,0 +1,58 @@
+import { DialogContentText } from "@material-ui/core";
+import ActionDialog from "@saleor/components/ActionDialog";
+import { ConfirmButtonTransitionState } from "@saleor/components/ConfirmButton";
+import React from "react";
+import { defineMessages, useIntl } from "react-intl";
+
+const messages = defineMessages({
```

Comment:
Move to local mssages file with named export, e.g `productVariantEndPreorderDialogMessages`

Reply:
done